test/public/overview.test.js

```diff
@@ -53,19 +52,21 @@ module.exports = function () {
         });
 
         pti.write([...jsCoverage, ...cssCoverage]);
+        await server.close();
```

Comment:
Which one has to close first? The browser or the server, I would assume browser.

Reply:
The chronology does not matter regarding the testing.